Tests/js/views/assets/ez-dashboardblocksview-tests.js

```diff
@@ -15,41 +15,42 @@ YUI.add('ez-dashboardblocksview-tests', function (Y) {
 
     Y.eZ.DashboardBlockBaseView = Y.Base.create('dashboardBlockBaseView', Y.View, [], {}, {
         ATTRS: {
-           priority: {},
+            priority: {},
             identifier: {}
         }
     });
 
+    Y.eZ.DashboardBlockAllContentView = Y.eZ.DashboardBlockBaseView;
+
     renderTest = new Y.Test.Case({
         name: 'eZ Dashboard Blocks View render test',
 
         setUp: function () {
-            this.view = new View(VIEW_CONFIG);
+            this.view = new View(Y.merge(VIEW_CONFIG, {rootLocation: new Y.Base()}));
         },
 
         tearDown: function () {
             this.view.destroy();
         },
 
-        'Should render view correctly': function () {
+        'Should render view': function () {
             var view = this.view,
                 templateCalled = false,
-                origTpl = view.template;
+                origTpl = view.template,
+                block = this.view.get('blocks')[0];
 
             view.template = function () {
                 templateCalled = true;
 
                 return origTpl.apply(this, arguments);
             };
 
-            view.addBlock(new Y.eZ.DashboardBlockBaseView({
-                priority: 1,
-                identifier: 'A'
-            }));
             this.view.render();
 
             Y.Assert.isTrue(templateCalled, 'The template should have been used to render view');
             Y.Assert.areSame(1, view.get('container').one(SELECTOR_CONTENT).get('children').size(), 'Should render one block inside');
+            Y.Assert.areSame(view, block.getTargets()[0], 'The view should be a block events target');
```

Comment:
this is completely unrelated to the `render` method, this should be tested in a test case dedicated to the `blocks` attribute default value.